brightener.h

```diff
@@ -1,10 +1,15 @@
 #pragma once
 #include <cstdint>
+#include <malloc.h>
 
 struct Image {
-	int rows;
-	int columns;
-	uint8_t pixels[1024 * 1024]; // max 1k x 1k image
+	int rows{};
+	int columns{};
+	//uint8_t pixels[1024 * 1024]; // max 1k x 1k image
+	uint8_t* pixels = (uint8_t*)malloc(1024 * 1024);
+
+	//std::unique_ptr<uint8_t> pixels(new uint8_t[1024*1024]);
```

Comment:
Not successful in using unique_ptr? Maybe because it's templated as uint_8 but you're giving it an array (pointer) of that type?